javalib/src/main/scala/java/io/StringReader.scala

```diff
@@ -23,6 +23,7 @@ class StringReader(s: String) extends Reader {
   }
 
   override def mark(readAheadLimit: Int): Unit = {
+    if (readAheadLimit < 0) throw new IllegalArgumentException("Read-ahead limit < 0")
```

Comment:
The `throw` should be on a new line, indented 2 spaces, because the `if` is in statement position. See https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#ifelse.

Reply:
Fixed